Homework10/GenericArray.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Homework10
+{
+    public class GenericArray<T> where T : Human
+    {
+        private T[] _array;
+
+        public GenericArray(int size)
+        {
+            _array = new T[size + 1];
+        }
+
+        public int GetAraryLength() => _array.Length;
+
+        public void AddElement(int index, T element)
+        {
+            _array[index] = element;
+        }
+
+        public T GetElement(int index) => _array[index];
+
+        public void RemoveElement(ref T[] _array, int index)
```

Comment:
Naming rule for parameters - starts with lower case letter

Reply:
fixed